src/unstract/sdk/embedding.py

```diff
@@ -42,12 +64,25 @@ def get_embedding(self, adapter_instance_id: str) -> BaseEmbedding:
             embedding_adapter_class = embedding_adapter(embedding_metadata)
             return embedding_adapter_class.get_embedding_instance()
         except Exception as e:
```

Comment:
@gaya3-zipstack  are we catching the adapater error here?

Reply:
Yes any AdapterError should be caught here @johnyrahul

Reply:
NIT: @gaya3-zipstack it'll be good if we can throw a specific error from the adapter and catch it here. Otherwise we might end up exposing some pythonic error to the user